storybook/preview.js

```diff
@@ -148,4 +165,72 @@ export const parameters = {
     },
 };
 
-export const decorators = [swcThemeDecorator];
+export const decorators = [
```

Comment:
how is this different from the projects/story-decorator? can we have one or the other or do we need both?